app/services/copy_course.rb

```diff
@@ -24,7 +26,7 @@ def make_copy
   def copy_main_course_data
     # Extract the attributes we want to copy
     params_to_copy = %w[school title term description start end subject slug timeline_start
-                        timeline_end type flags]
+                        timeline_end type flags created_at weekdays]
```

Comment:
I don't think we want to copy `created_at`. Better to use the timestamp from when the copying happened.

Reply:
@ragesoss
Sure, I have removed `created_at`. Now, it will automatically take the current date when the copy of the course is made.